lib/cadet/jobs/autograder/lambda_worker.ex

```diff
@@ -0,0 +1,99 @@
+defmodule Cadet.Autograder.LambdaWorker do
+  @moduledoc """
+  This module submits the answer to the autograder and creates a job for the ResultStoreWorker to
+  write it to db on success.
+  """
+  use Que.Worker, concurrency: 20
+
+  require Logger
+
+  alias Cadet.Autograder.ResultStoreWorker
+
+  @lambda_name :cadet |> Application.fetch_env!(:autograder) |> Keyword.get(:lambda_name)
+
+  def build_request_params(%{question: question, answer: answer}) do
```

Comment:
yep

Reply:
To elaborate, parsing of response can be tested based on what goes into the next job (from stubbed results) but testing request params similarly with a mock isn't as straightforward because of all the metadata.